src/lib.rs

```diff
@@ -101,7 +101,12 @@ use std::path::{Path, PathBuf};
 use std::result;
 use std::vec;
 
-pub use same_file::is_same_file;
```

Comment:
Should we be able to apply the `deprecated` attribute to this re-export? Is there a reason not to?